### PR TITLE
Add VCU118 Support to mp/pulp-v1-os-fpga Branch

### PR DESCRIPTION
**Pull request by mrbilandi**
This pull request integrates support for the Xilinx VCU118 FPGA platform into the mp/pulp-v1-os-fpga branch.

### Changes
cheshire/Makefile:

Modified to support any specified platform using BOARD=<desired-board>. For example, use "make ara-chs-xilinx BOARD=vcu118" to build for the VCU118 platform.

## Checklist

- [ ] Automated tests pass
- [ ] Changelog updated
- [ ] Code style guideline is observed

### CODE REVIEW

**Comment by aitesam961** [comment]
Nice work @mrbilandi were you able to run baremetal or OS on it?

File: Makefile [hunk 1]
```diff
@@ -40,8 +40,8 @@ CXX    = g++
 endif
 
 # We need a recent LLVM to compile Verilator
-CLANG_CC  ?= clang
-CLANG_CXX ?= clang++
+CLANG_CC  ?= clang-14
+CLANG_CXX ?= clang++-14
```
Comment (mp-17) [inline Makefile:44]: We can drop this change, right?

**Review by mp-17 — CHANGES_REQUESTED** [review]
Thanks a lot! Once the questions are addressed, we can merge.

**Comment by mrbilandi** [comment]
Hi @aitesam961 
Yes, I was able to run. I think Matteo has plans to release the instructions.

**Comment by mp-17** [comment]
Hey @mrbilandi,

Since the base branch went out of sync due to flow modifications, I have pushed there your (adapted) commit and authored it with your username and email.

Everything is [here](https://github.com/pulp-platform/ara/pull/319/commits/7747bb0ab1a7540be87c1a13e25395f4940892da) in https://github.com/pulp-platform/ara/pull/319.

Let me know if this is okay for you and if I need to re-open this PR ;-)

**Pull request closed without merging**